docs/network/architecture.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Etherlink architecture
+---
+
+Etherlink's main components are its nodes and the sequencer.
+
+## Sequencer
+
+Etherlink relies on a sequencer to process transactions in a fair order.
```

Comment:
I would drop the part about fair. The point of the sequencer is to provide lower latency compared with having to wait for transactions to be posted on L1